test/integration/user_can_view_past_orders_test.rb

```diff
@@ -8,7 +8,7 @@ class UserCanViewPastOrdersTest < ActionDispatch::IntegrationTest
     visit '/orders'
 
     assert page.has_content?('Your Orders')
-    assert page.has_content?('ordered')
+    assert page.has_content?('Pending')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.